### PR TITLE
3dmol.js minified files and configuration

### PR DESCRIPTION
**Pull request by dkoes**
3Dmol.js is a javascript library for molecular visualization with WebGL.  It is licensed under a BSD license.

### CODE REVIEW

**Comment by dkoes** [comment]
The code was minimized using the google closure compiler.  It sure isn't readable.

**Comment by megawac** [comment]
Don't mind @jsdelivrbot, he's pretty dumb

File: files/3dmol.js/info.ini [hunk 1]
```diff
@@ -0,0 +1,5 @@
+author = "David Koes"
+github = "https://github.com/dkoes/3Dmol.js"
+homepage = "http://3dmol.csb.pitt.edu"
+description = "A modern, object-oriented JavaScript library for visualizing molecular data."
+mainfile = "3Dmol-min.js"
```
Comment (megawac) [inline files/3dmol.js/info.ini]: I would make this the nojquery version personally

**Comment by megawac** [comment]
Also @dkoes you'lll need to start tagging releases with git before we accept this pr

**Comment by megawac** [comment]
+ the version in your package.json is out of date

**Comment by dkoes** [comment]
Release tagged and package.json updated.

**Comment by megawac** [comment]
Cool, for that auto updating to work you have to publish tags on your repo @dkoes 

Thanks!

**Pull request merged**